tests/components/recorder/test_statistics.py

```diff
@@ -1733,14 +1733,19 @@ def set_state(entity_id, state, **kwargs):
         states[mp].append(
             set_state(mp, "idle", attributes={"media_title": str(sentinel.mt1)})
         )
-        states[mp].append(
-            set_state(mp, "YouTube", attributes={"media_title": str(sentinel.mt2)})
-        )
         states[sns1].append(set_state(sns1, "10", attributes=sns1_attr))
         states[sns2].append(set_state(sns2, "10", attributes=sns2_attr))
         states[sns3].append(set_state(sns3, "10", attributes=sns3_attr))
         states[sns4].append(set_state(sns4, "10", attributes=sns4_attr))
 
+    with patch(
```

Comment:
There were two states behind saved with the exact same microsecond which isn't actually possible in production.  Postgresql will return them in a different order so the test was failing.  I moved one of the states a microsecond later to avoid the issue.